tests/Main.hs

```diff
@@ -696,8 +696,78 @@ runClashTest = defaultMain $ clashTestRoot
         , runTest "BlockRamTest" def{hdlSim=False}
         , runTest "Compression" def
         , runTest "DelayedReset" def
-        , runTest "DualBlockRam0" def{verilate=SimOnly}
-        , runTest "DualBlockRam1" def{verilate=SimOnly}
+        , runTest "DualBlockRam" def
+          { clashFlags = ["-main-is", "testBench_WF_WF_AB"]
+          , buildTargets = BuildSpecific ["testBench_WF_WF_AB"]
+          , verilate=SimOnly}
```

Comment:
That's quite interesting, that you need both `BuildSpecific` and `-main-is`. Is this is a bug in the `LoadModules` stuff? I'll investigate.

What's wrong with verilator?